karim.html

```diff
@@ -271,12 +271,14 @@
             var checkbox = todo[i];
             var data = checkbox.parentNode.dataset;
 
-            firebase.database().ref('orders/' + data.reference + '/' + data.uid).set({
-              name: data.name,
-              email: data.email,
-              ticketName: data.ticketName,
-              invoice: checkbox.checked ? 'SENT' : 'REQUESTED'
-            });
+            if (data.ticketName === 'External') {
```

Comment:
We only update the reference if it is an External ticket, because only these are actually shown as checkboxes.